File: src/main/java/it/contrader/dto/WishListDTO.java
package it.contrader.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class WishListDTO {
	
	private Long id;
	
	private String name;
	
	private String description;
	
	private UserDTO proprietario;

}

File: src/dto/wishlistdto.ts
import { UserDTO } from './userdto';

export class WishListDTO{
    id: number;

    name: string;

    description: string;

    proprietario_id: number;

    constructor(id?: number, name?: string){
        this.id = id;
        this.name = name;
    }
   
    

}